ThunderCloud/StreamingContentController.swift

```diff
@@ -299,10 +305,11 @@ class StreamingContentFileOperation: CustomOperationBase {
             return
         }
         
-        fileRequestController.downloadFile(withPath: fileDownloadURLString, progress: { (progress: CGFloat, totalBytes: Int, bytesTransferred: Int) in
-            
-        }) { (fileLocation: URL?, downloadError: Error?) in
-            
+        guard let fileRequestController = fileRequestController else {
+            return
+        }
+        
+        fileRequestController.download("", progress: nil) { (response, fileLocation, error) in
```

Comment:
Is the first parameter of this meant to be empty?

Reply:
Aye, the full URL to the file is passed into the init method of `StreamingContentFileOperation` so we just provide an empty string here! Will comment that up :)